src/Source.cpp

```diff
@@ -8,6 +8,11 @@
 // Screaming Udder!                              https://esss.se
 
 #include "Source.h"
+#include "helper.h"
+#include "logger.h"
+#include <chrono>
+#include <fstream>
+#include <thread>
```

Comment:
Why have all these includes appeared? It doesn't look like any code was added which need them.

Reply:
I do not know but it is not entirely unlikely that the reason is something similar to the comment above this one. It has now been fixed.